web/vtadmin/scripts/build-pscale-vtadmin.ts

```diff
@@ -0,0 +1,29 @@
+const rewire = require('rewire');
```

Comment:
This file, and all files in its directory `scripts` is only ever used by planetscale to release @planetscale/vtadmin npm package.